### PR TITLE
Move resources to us-west-2

### PR DESCRIPTION
**Pull request by blucas**
Additionally, prune all resources on `remove` operation,  except for `prod` stage

### CODE REVIEW

File: sst.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "name": "pickup",
-  "region": "us-east-2",
+  "region": "us-west-2",
```
Comment (olizilla) [inline sst.json:3]: note: this only applies to local dev deploys run with `npm start` or `npx sst start`. The region is configured for staging and PR builds and prod in seed.run settings.

File: stacks/index.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
-import { Tags } from 'aws-cdk-lib'
+import { Tags, RemovalPolicy } from 'aws-cdk-lib'
 import { PickupStack } from './PickupStack'
 import { BasicApiStack } from './BasicApiStack'
 import { App } from '@serverless-stack/resources'
 
 export default function (app: App): void {
+  if (app.stage !== "prod") {
+    app.setDefaultRemovalPolicy(RemovalPolicy.DESTROY)
```
Comment (olizilla) [inline stacks/index.ts:8]: it's been useful to have `staging` not auto destroy things so we can see when we need to do a migration... the deploy will fail if the db needs a migration for example... 

as i understand it, sst destroys stateless resources like lambdas automatically by default when it needs to replace them, but will not do so for buckets and tables unless this removalPolicy default is changed.

https://docs.sst.dev/advanced/removal-policy#retained-resources

File: cdk.context.json [hunk 3]
```diff
@@ -6,5 +6,16 @@
     "us-east-1d",
     "us-east-1e",
     "us-east-1f"
+  ],
```
Comment (olizilla) [inline cdk.context.json:9]: not clear if we need this file. I think it should probably be in the .gitignore (sorry). I think it only applies to deployments done locally and wouldn't really make sense if used for staging or prod from seed.run CD.

File: sst.json (hunk 1 above)
Comment (blucas) [reply sst.json:3]: Seed doesn't utilize this file? Bummer :(

File: stacks/index.ts (hunk 2 above)
Comment (blucas) [reply stacks/index.ts:8]: I added this block as we have quite a few PR environments which have left-over dynamo, s3, etc related resources. Would it make sense to slightly modify the condition to:
```javascript
  if (!(app.stage in ['prod', 'staging'])) {
    app.setDefaultRemovalPolicy(RemovalPolicy.DESTROY)
  }
```

File: cdk.context.json (hunk 3 above)
Comment (blucas) [reply cdk.context.json:9]: I'll remove the file and create another PR to update .gitignore

**Review by olizilla — APPROVED** [review]

**Pull request merged**